### ISSUE TITLE
Cannot Connect to Proxy Server Error

### ISSUE THREAD

**Issue by crashxpilot** [kakaroto/SWProxy#188]
Proxy server appears to be down (hopefully this is just a minor maintenance). But I tried reinstalling the SW app altogether lol. Tried on two devices, tried reinstalling the parser too. 

Hope this gets fixed :(

**Comment by kakaroto** [comment]
There is no "maintenance", you are running the proxy server yourself on your machine, not us.
Make sure the firewall is disabled, try using a port other than 8080, etc.. (tips are in the readme file).

**Comment by crashxpilot** [comment]
Hi thanks for the reply and sorry for my misuse of some terms. I'm not too technical as you are obviously.

The error I'm getting on my mobile device reads: err_proxy_connection_failed. Attaching a screenshot below. I happen to check the proxy.txt file created during the parsing process, I believe, and it reads:

proxy - ERROR - <ProxyConnectionFailed - ipv6.msftconnecttest.com:80 - gaierror(11001, 'getaddrinfo failed')>
Traceback (most recent call last):
  File "proxy.py", line 471, in _process_rlist
  File "proxy.py", line 395, in _process_request
ProxyConnectionFailed: <ProxyConnectionFailed - ipv6.msftconnecttest.com:80 - gaierror(11001, 'getaddrinfo failed')>
proxy - INFO - 192.168.254.4:51105 - GET ipv6.msftconnecttest.com:80/connecttest.txt - None None - 0 bytes

I tried resetting browser settings, cleared cache ofc, tried manually using proxy on the computer itself, to no avail. I did turn off Windows firewall, turned off my AV and all other protection I could think of. Even tried uninstalling this CCleaner. I tried manually allowing swproxy.exe on the Windows firewall setup, same thing.

I'm really running out of options. I just wish this gets fixed. Any suggestions?

![err_proxy_connection failed](https://cloud.githubusercontent.com/assets/28673847/26130344/cf65d574-3ac6-11e7-9055-c6c1291ff604.png)

**Comment by kakaroto** [comment]
Did you try setting a port different from 8080 in the swproxy app before clicking start ? try that and make sure you're on the same network as from your phone and from your pc.

**Comment by crashxpilot** [comment]
Hi, I did try other ports. Though I'm not really sure what ports I could use, but I tried ports 8000 and 9000. Also, pc and phone are in same network. I even tried deleting 'proxy' items from the registry, following this link https://www.youtube.com/watch?v=fU6jiUv_wyM.

The thing is this used to work on this same pc on the same network. I'll try system restore once I got the chance.

Thanks again though.
